saskatoon/harvest/admin.py

```diff
@@ -136,27 +139,34 @@ def reset_authorize(self, request, queryset):
         messages.add_message(request, messages.SUCCESS,
                              "Successfully reset authorizations for this season")
 
-    @admin.action(description="Create missing auth users using pending email")
+    @admin.action(description="Create missing auth users using pending or comments email")
     def create_owner_user(self, request, queryset):
         """Create new AuthUser objects for owners (Persons) without email address"""
+
         qs = queryset.filter(owner__organization__isnull=True,
                              owner__person__isnull=False,
-                             owner__person__auth_user__isnull=True,
-                             pending_contact_email__isnull=False)
-
+                             owner__person__auth_user__isnull=True)
         nb_users = 0
         for _property in qs:
             try:
-                AuthUser.objects.create(email=_property.pending_contact_email,
-                                        person=_property.owner.person)
-                nb_users += 1
+                email = _property.pending_contact_email
+                if not email:
+                    emails = _property.owner.person.comment_emails
+                    if len(emails) > 1:
+                        messages.add_message(request, messages.WARNING,
+                                f"{_property} has multiple emails in comments")
+                        continue
+                    elif len(emails) == 1:
+                        email = emails[0]
```

Comment:
@tristanlatr i suggest we skip cases where there is more than one email and check them one by one